api4/oauth.go

```diff
@@ -67,15 +82,14 @@ func updateOAuthApp(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.SetInvalidParam("oauth_app")
 		return
 	}
+	auditRec.AddMeta("oauth_app_name", oauthApp.Name)
 
 	// The app being updated in the payload must be the same one as indicated in the URL.
 	if oauthApp.Id != c.Params.AppId {
 		c.SetInvalidParam("app_id")
 		return
 	}
 
-	c.LogAudit("attempt")
```

Comment:
Was this line removed intentionally?

Reply:
It was simply moved further up the method to live with the new audit stuff.